.github/workflows/enforce-changelog-entry.yaml

```diff
@@ -0,0 +1,11 @@
+name: "Enforce changelog entry"
+on:
+  pull_request:
```

Comment:
**Reminder:** We need to skip this for dependabot somehow.

Reply:
In 0e9dbf5398365fdb7daece91ba69501ea372375a I extended the labels to now skip on `dependencies` (that is used by dependabot by default) and `tests` (that we use)